backend/app/bug_handler.go

```diff
@@ -0,0 +1,194 @@
+package app
+
+import (
+	"errors"
+
+	"github.com/codescalersinternships/Flyspray/models"
+	"github.com/gin-gonic/gin"
+	"github.com/go-playground/validator/v10"
+	"github.com/rs/zerolog/log"
+	"gorm.io/gorm"
+)
+
+type createBugInput struct {
+	Summary     string `json:"summary"`
+	ComponentID int    `json:"component_id" validate:"required"`
```

Comment:
the rest of the Bug fields should be included in user input too. Like severity, category, ..etc